activesupport/test/evented_file_update_checker_test.rb

```diff
@@ -36,6 +36,8 @@ def rm_f(files)
   end
 
   test 'notifies forked processes' do
+    skip if defined?(JRUBY_VERSION)
```

Comment:
Looks like it got merged. Want another PR or can someone just patch up that line for me? :-)